hack/triage/config.yaml

```diff
@@ -0,0 +1,269 @@
+    type: issue
+    filters:
+      - label: "!triage/accepted"
+
+  # Phase: Actionable
+
+  actionable-not-assigned-without-help-or-good-first-issue:
+    # WHY?: issue actionable must be assigned to someone or seeking for help
+    name: "Issues not assigned without help or good-first-issue"
+    resolution: "Apply /help or /good-first-issue"
+    type: issue
+    filters:
+      - label: "triage/accepted"
+      - label: "!(help|good-first-issue)"
+      - tag: "!assigned"
+
+  actionable-updated:
+    # WHY?: maintainers are supposed to keep up with issue discussions
+    name: "Issues with updates in the last month"
+    resolution: "Check updates"
+    type: issue
+    filters:
+      - label: "triage/accepted"
+      - commented: -30d
+      - tag: "!member-last"
+
+  actionable-all:
+    name: "All the actionable issues"
+    type: issue
+    filters:
+      - label: "triage/accepted"
+
+  # Fix-me
+  # WHY?: Let's try to be diligent in using labels
+
+  fixme-avoid-triage-needs-information:
+    name: "With triage/needs-information (let's not use this label)"
+    resolution: "Remove triage/needs-information, se priority/awaiting-more-evidence instead"
+    type: issue
+    filters:
+      - label: "triage/needs-information"
+
+  fixme-avoid-kind-design:
+    name: "With kind/design (let's not use this label)"
+    resolution: "remove kind/design, use kind/proposal instead"
+    type: issue
+    filters:
+      - label: "kind/design"
+
+  fixme-kind-proposal-without-kind-feature:
+    name: "With kind/proposal without kind/feature"
+    resolution: "Add kind/feature (we usually write proposals for new features, so kind/proposal and kind/feature should go together)"
+    type: issue
+    filters:
+      - label: "kind/proposal"
+      - label: "!kind/feature"
+
+  fixme-kind-support-with-other-kinds:
+    name: "With kind/support and also other kinds"
+    resolution: "Remove incorrect kinds (kind/support should not be combined with other kinds)"
+    type: issue
+    filters:
+      - label: "kind/support"
+      - label: "kind/(feature|documentation|bug|flake|cleanup|design|proposal|deprecation|regression|api-change|failing-test|release-blocking)"
+
+  fixme-triage-accepted-with-kind-support-or-priority-awaiting-more-evidence:
+    name: "With triage/accepted and kind/support or priority/awaiting-more-evidence"
+    resolution: "Remove triage/accepted (triage/accepted should not be applied to issue with kind/support or priority/awaiting-more-evidence)"
+    type: issue
+    filters:
+      - label: "triage/accepted"
+      - label: "(kind/support|priority/awaiting-more-evidence)"
+
+  fixme-help-or-good-first-issue-before-triage-accepted:
+    name: "With help or good-first-issue applied before triage/accepted"
+    resolution: "help or good-first-issue should not be applied before triage/accepted"
+    type: issue
+    filters:
+      - label: "(help|good-first-issue)"
```

Comment:
Does this work? The label names are "help wanted" and "good first issue" as far as I can tell (probably also in other rules)

Reply:
Fixed